server/meeting.go

```diff
@@ -92,8 +103,10 @@ func (p *Plugin) GenerateHashtag(channelID string, nextWeek bool, weekday int) (
 		prefix = matchGroups[1]
 		hashtagFormat = strings.TrimSpace(matchGroups[2])
 		postfix = matchGroups[3]
+		formattedDate := meetingDate.Format(hashtagFormat)
+		formattedDate = strings.ReplaceAll(formattedDate, " ", "_")
```

Comment:
Why is this necessary? What about other formats that use a delimiter other than spaces and underscores?

Reply:
Considering `-` breaks on certain DB's,  `_` is the only safe character. But since `Format` which our plugin heavily rely on has special meaning `_`. I treated string before and after `Format`.